Controllers/HotelRoomController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace AsyncInn.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
This is where we want to say this whole controller lives under `/api/Hotels/{hotelId}/Rooms`.